src/utils/remove-notification.ts

```diff
@@ -14,6 +17,9 @@ export const removeNotification = (
     return notifications;
   }
 
+  const notificationRow = document.getElementById(notification.id);
+  notificationRow.className += ` ${ANIMATE_NOTIFICATION_CLASS_NAME}`;
```

Comment:
The thing is that this breaks the animation though. If you use gitify without this setting the animation is now instant and you don't see it leaving. The animation needs the animation time before removing the notification.

I suppose what we can do is add a delay in this file if `delayNotificationState` is not enabled. We already have access to the setting state!

Reply:
Good point.

Update pushed.  I did notice that the animations are pretty quick ⚡

Reply:
Checking it out atm, but like the approach code wise!

Reply:
Any further changes or good to roll?

Reply:
Sorry, yes, good to go! I've had a few visual anomalies, but I don't think they're related, it is just webpack or something from running locally.